### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wrathwolf**
Update wiki and mailing list info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ The sub project has no (pre)releases yet, work in progress is within the main br
...
     * [Registration/Join] (Meeting link tbd)
     * Schedule: tbd
-    * Minutes: Access [meeting minutes] (update this link to your wiki page using the "Share" link).
+    * Minutes: Access [meeting minutes](https://wiki.camaraproject.org/x/sACeAQ)
 * Mailing List
     * (NOTE: mailing list need still to be created - thanks for your patience)
-    * Subscribe / Unsubscribe to the mailing list of this Sub Project <https://lists.camaraproject.org/g/sp-location-insights>.
-    * A message to the community of this Sub Project can be sent using <sp-location-insights@lists.camaraproject.org>.
+    * Subscribe / Unsubscribe to the mailing list of this Sub Project <https://lists.camaraproject.org/g/sp-locationinsights>.
+    * A message to the community of this Sub Project can be sent using <sp-locationinsights@lists.camaraproject.org>.
```
Comment (hdamker) [inline README.md:36]: I would propose to follow the same syntax as for the GitHub teams and other mailing lists and transform the CamelCase "LocationInsigts" to kebab-case "location-insights" within the mailing list (as originally proposed).

File: README.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ The sub project has no (pre)releases yet, work in progress is within the main br
 * Meetings
     * [Registration/Join] (Meeting link tbd)
     * Schedule: tbd
-    * Minutes: Access [meeting minutes] (update this link to your wiki page using the "Share" link).
+    * Minutes: Access [meeting minutes](https://wiki.camaraproject.org/x/sACeAQ)
 * Mailing List
     * (NOTE: mailing list need still to be created - thanks for your patience)
```
Comment (hdamker) [inline README.md:34]: This line need to be deleted.

**Review by hdamker — CHANGES_REQUESTED** [review]

**Review by hdamker — APPROVED** [review]
Mailing list is now as planned sp-location-insights@lists.camaraproject.org

**Comment by hdamker** [comment]
@camaraproject/location-insights_codeowners @camaraproject/location-insights_maintainers:
The mailing list https://lists.camaraproject.org/g/sp-location-insights is now available.

Please 
a) subscribe to the mailing list
b) approve and merge the PR

**Comment by hdamker** [comment]
@camaraproject/location-insights_codeowners can one of the Codeowners please approve this PR so that it can be merged? The last change was done three weeks ago ...

**Review by chinaunicomyangfan — APPROVED** [review]
LGTM

**Pull request merged**